app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,9 @@
  *= require_tree .
  *= require_self
  */
+
+html {
+  background-image: url("dark_embroidery.png");
```

Comment:
One thing that did work for another student to get the background image to display was:
`background-image: image-url("....");`